data/schema.js

```diff
@@ -0,0 +1,23 @@
+import { buildSchema } from 'graphql';
+import { GraphQLList } from 'graphql';
```

Comment:
Since these are both from same source I believe you can comma delimit instead of putting on separate lines.